src/components/accordion/accordion.vue

```diff
@@ -4,8 +4,8 @@
                 class="u-reset ui-accordion--head"
                 role="heading"
                 tabindex="0"
-                v-if="!!$scopedSlots.head"
                 :aria-expanded="state.isOpen ? 'true' : 'false'"
+                v-if="!!$scopedSlots.head"
```

Comment:
Isn't this against the stylelint rules within our projects?

Reply:
You might be right, but the vue eslint plugin isn't smart enough to catch it. I've fixed it anyway!